packages/tldraw/src/lib/utils/assets/assets.ts

```diff
@@ -60,7 +60,7 @@ export async function downsizeImage(
 	opts = {} as { type?: string; quality?: number }
 ): Promise<Blob> {
 	const image = await MediaHelpers.usingObjectURL(blob, MediaHelpers.loadImage)
-	const { type = blob.type, quality = 0.92 } = opts
```

Comment:
as long as i'm in here looking at this: this brings this more in line with the Cloudflare default of 85 (not that this code is actually used anywhere at the moment, to be clear)